doc/rados/operations/health-checks.rst

```diff
@@ -25,6 +25,15 @@ Definitions
 Monitor
 -------
 
+DAEMON_OLD_VERSION
+__________________
+
+Warn if old version(s) of Ceph are running on any deamons.
+This will be an expected warning while in the middle of
```

Comment:
we should be able to avoid this during an upgrade, e.g. only warn after a certain period of time. users shouldn't need to take any action in the majority of cases

Reply:
I think we talked about having the upgrade recipes to do something to mark upgrade in progress and clear when upgrade complete.  However, for a manual update my doc would be appropriate.  Can we leave this to a follow on fix when the different recipes can be changed?

Reply:
yes that could be addressed in a follow-up, just want to make sure we fix it up before pacific is released

Reply:
@dzafman I'd expect this warning to catch intentional or unintentional incomplete upgrades, by muting the warning during upgrades, we essentially lose its purpose. I'd like this warning to trigger after a reasonable amount of time, we can go with a safe default.

Reply:
I'm not convinced there is a safe default for the trigger delay that would work because of the variation in cluster sizes.

In order for teuthology upgrade tests to work I'm making cephadm mute DAE_OLD_VERSION warnings at the start of upgrade and unmute after upgrade completion.  If you pause upgrade, I'm thinking of leaving the warning muted, but I'm open to change that.  We could unmute on pause and mute on resume.

The documentation text above would be changed to "Unless you use cephadm to upgrade, this will be an expected warning while in the middle of an upgrade."  The plan is to the the mute/unmute other upgrade handlers like Ansible and update the doc accordingly.